grc/gui/DrawingArea.py

```diff
@@ -210,6 +212,54 @@ def draw(self, widget, cr):
         cr.rectangle(0, 0, width, height)
         cr.fill()
 
+        # Probably not the most efficient
+        if Actions.TOGGLE_SHOW_FLOWGRAPH_GRID.get_active():
+            grid_mode = Gtk.Application.get_default().config.flowgraph_grid_mode()
+            cr.set_source_rgba(*FLOWGRAPH_GRID_COLOR)
+
+            # TODO: Could optimize this. These only need to be recalculated if adjusting the canvas size
+            grid_size = Constants.CANVAS_GRID_SIZE
+            points_w = range(0, int(width), grid_size)
+
+            grid_offset = int(grid_size/2)
+            points_h = range(grid_offset, int(height)+grid_offset, grid_size)
+
+            if grid_mode == "lines":
+                cr.set_line_width(1.0)
+            
+                for w_i in points_w:
+                    cr.move_to(w_i, 0)
+                    cr.line_to(w_i, height)
+                for h_i in points_h:
+                    cr.move_to(0, h_i)
+                    cr.line_to(width, h_i)
+                    cr.stroke()
+            # elif grid_mode == "dots":
```

Comment:
I think we should either leave this in, or delete these lines. I'm for leaving it in, even if it's not obvious how to change it (I don't think this PR needs to add ways to set this through the GUI, for example).